applications/asset_tracker_v2/tests/debug_module/prj.conf

```diff
@@ -6,3 +6,6 @@
 
 CONFIG_UNITY=y
 CONFIG_HEAP_MEM_POOL_SIZE=1024
+
+#Need to enable Event manager to make variable CONFIG_EVENT_MANAGER_MAX_EVENT_CNT visible.
```

Comment:
?
Why adding this comment? It seems important to you only because you focus on it now. If you think of it you could easily add similar comments to all options.

If it's needed then the comment is misleading. You don't need the variable to be visible, you most probably need the macro to be defined. Please also add space.
```
# Make CONFIG_EVENT_MANAGER_MAX_EVENT_CNT defined
CONFIG_EVENT_MANAGER=y
```